addons/website_crm_partner_assign/views/website_crm_partner_assign_templates.xml

```diff
@@ -138,6 +138,11 @@
                 <h5>No results found for "<span t-out="searches.get('search', '')"/>"</h5>
                 <a href="/partners">See all resellers</a>
             </div>
+            <div t-elif="fallback_all_countries" class="mt-4 alert alert-primary alert-dismissible fade show" role="alert">
+                <i class="fa fa-info-circle me-2"/>
+                There is no matching partners found for the selected country. Displaying results across all countries instead.
```

Comment:
"There is no matching partners" -> is is singular and partners plural ? Is it valid ?